README.md

```diff
@@ -31,3 +39,17 @@ Unit tests are located under test/. Run the entire test suite with
 pytest
 ```
 or test individual files via, e.g., `pytest test/test_data.py`
+
+## Cite
+
+Please cite [our paper](https://papers.nips.cc/paper_files/paper/2023/hash/8ca113d122584f12a6727341aaf58887-Abstract-Conference.html) if you use this code in your own work:
+
+```bibtex
+@inproceedings{
+    azabou2023unified,
+    title={A Unified, Scalable Framework for Neural Population Decoding},
+    author={Mehdi Azabou and Vinam Arora and Venkataramana Ganesh and Ximeng Mao and Santosh Nachimuthu and Michael Mendelson and Blake Richards and Matthew Perich and Guillaume Lajoie and Eva L. Dyer},
+    booktitle={Thirty-seventh Conference on Neural Information Processing Systems},
+    year={2023},
+}
+```
```

Comment:
idk if this should go in temporaldata? you sure?

Reply:
temporarily until we make a publication for this. we will update.